### PR TITLE
[4] correct property type

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ class PlgSampledataBlog extends CMSPlugin
 	/**
 	 * Holds the menuitem model
 	 *
-	 * @var    MenusModelItem
+	 * @var    \Joomla\Component\Menus\Administrator\Model\ItemModel
```
Comment (wilsonge) [inline plugins/sampledata/blog/blog.php]: Can we import this please as it's also used to initialise the class below in the actual code

Comment (PhilETaylor) [reply plugins/sampledata/blog/blog.php]: lol you told me off for importing previously and wanted them all long hand :)

Comment (wilsonge) [reply plugins/sampledata/blog/blog.php]: If it's just a code comment with no concrete uses in the file I prefer inline. If it's used as a concrete class in the code I prefer an import.

Is that clear or just confusing more? (Also FYI this is just my personal preference and isn't an official position taken by Joomla etc etc)

Comment (PhilETaylor) [reply plugins/sampledata/blog/blog.php]: imported

**Comment by wilsonge** [comment]
Thankyou and sorry for being a pain :)

**Pull request merged**